test/jdk/java/awt/List/KeyEventsTest/KeyEventsTest.java

```diff
@@ -50,6 +39,18 @@
 
 import jdk.test.lib.Platform;
 
+/*
+*  @test
+*  @key headful
```

Comment:
```suggestion
/*
 *  @test
 *  @key headful
```
Align asterisks to the asterisk on the opening line.